api/src/main/resources/metadata/Custom_Concepts.xml

```diff
@@ -1309,6 +1309,7 @@
     <concept_name  concept_name_id="159993"  concept_id="164988"  name="Restarted IAC"  locale="en"  locale_preferred="1"  creator="1"  date_created="2017-11-23 12:45:43"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="159993BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"/>
     <concept_name  concept_name_id="159992"  concept_id="164987"  name="Serum Crag Screening"  locale="en"  locale_preferred="1"  creator="1"  date_created="2017-11-23 12:44:15"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="159992BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"/>
     <concept_name  concept_name_id="159991"  concept_id="164986"  name="Serum Crag Screening Result"  locale="en"  locale_preferred="1"  creator="1"  date_created="2017-11-23 12:44:05"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="159991BBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"/>
+    <concept_name  concept_name_id="160179"  concept_id="165144"  name="Stable In DSDM"  locale="en"  locale_preferred="1"  creator="1"  date_created="2018-11-23 17:46:41"  concept_name_type="FULLY_SPECIFIED"  voided="0" uuid="6f6e2d05-abcb-4e5d-9e3c-172a64747e5d"/>
```

Comment:
@slubwama Stable in DSDM? This is not clear can you elaborate - as stable/not-stable are measures outside DSDM and applicable in general clinic settings

Reply:
@ssmusoke this is an assessment for a person whether they stable or unstable